components/notion/actions/retrieve-page-property-item/retrieve-page-property-item.mjs

```diff
@@ -0,0 +1,27 @@
+import notion from '../../notion.app.mjs';
+
+export default {
+  key: 'notion-retrieve-page-property-item',
+  name: 'Retrieve Page Property Item',
+  description:
+    'Retrieves a property_item object for a given page_id and property_id.',
+  version: '0.0.21',
```

Comment:
According to the linting instructions, I run `pnpm install` at the root of the project, run `pnpm install -r` to install the required dependencies, then I can run `npx eslint action/retrieve-page-property-item`? I'm new to using pnpm.

Reply:
Yeah, exactly. If you run `npx eslint action/retrieve-page-property-item --fix`, then it will automatically fix most of the linting rules, if not all of them

Reply:
I ran the eslint command and I don't see any changes made the to the component.  The version number occurred from testing the action in Pipedream several times, several different ways.  Is this an issue?

Reply:
Probably most of the linting issues are located in the app file, `notion.app.mjs`. For example, strings should use double quotes instead of single quotes.

Regarding the component version, this is known limitation. Sources support using the `pd dev` command which doesn't require updating the component version for every change and publish for testing, but actions don't support this yet. When publishing to the global registry (after merging to master), we apply the [versioning rules](https://pipedream.com/docs/components/guidelines/#versioning).

Reply:
fixed and added to branch